motr-notifier/src/main/java/uk/gov/dvsa/motr/notifier/notify/NotifySmsService.java

```
@@ -83,6 +101,15 @@ private SendSmsResponse sendSms(String phoneNumber, String smsTemplateId, Map<St
         return notificationClient.sendSms(smsTemplateId, phoneNumber, personalisation, "");
     }
 
+    private Map<String, String> getNotifyParameters(String body, Map<String, String> parameters) {
+        try {
+            return notifyTemplateEngine.getNotifyParameters(body, parameters);
+        } catch (NotifyTemplateEngineException exception) {
+            logger.error(exception.getMessage());
+            return Collections.emptyMap();
```

why suppress error? is it expected situation that getNotifyParameters will throw exception? how rest of application will behave?